src/complete.ts

```diff
@@ -65,9 +110,9 @@ export function completeFromSchema(schema: {[table: string]: readonly (string |
     if (empty && !context.explicit) return null
     let options = topOptions
     if (parent) {
-      let columns = byTable[parent]
```

Comment:
Renamed `columns` to `completions`, because this variable can no longer hold columns from `byTable`, but also tables from `bySchema`.